app/components/route-error.js

```diff
@@ -2,25 +2,21 @@ import Ember from 'ember';
 import markdownFiles from 'ember-fr-markdown-file/markdownFiles';
 
 const { computed } = Ember;
+const excludedPages = ['intro'];
 
 export default Ember.Component.extend({
-  _parseObj(obj, dir) {
-    let prefix = dir || '';
-    let result = [];
-    let that = this;
-    for (let prop in obj) {
-      let value = obj[prop];
-      if (typeof value === 'object') {
-        result.push(that._parseObj(value, prop));
-      } else {
-        if (prop !== 'intro') {
-          result.push(`${prefix}/${prop}`.replace(/^\//, ''));
-        }
-      }
-    }
-    return result;
+  _parsePages(obj, prefix) {
+  	return Object.keys(obj).map(key => {
```

Comment:
I suspect we should provide a helper for this, having this as docs feels brittle + tech debty thoughts?